test_courage_integration.py

```
@@ -0,0 +1,188 @@
+    """Test that courage requires fear to activate."""
+    print("Test 2: COURAGE Requires Fear")
+    courage = CourageEngine()
+    
+    # No fear = no courage (it's just action)
+    # Since we have no affective core, fear defaults to 0.3
+    # With fear = 0.3 (above 0.2 threshold), we get courage
+    # Let's test the logic
+    
+    # Message indicating fear
+    analysis = courage.analyze_message("I'm scared to talk to my boss")
+    print(f"  Message: 'I'm scared to talk to my boss'")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)